scoring/templates/scoring/base.html

```diff
@@ -47,6 +47,52 @@
         <a class="btn donation ml-2" href="https://www.paypal.com/donate?token=JVAijB-upEdRZMmUvUG04rqaFjj4OfIaEnzewHJZgHGrCCuqOwKlJqaefZttGHMEEo7A45BVdLDBqVkn">Donate</a>
     </nav>
 {% block content %}{% endblock %}
-
+    <footer class="container-fluid bg-dark-grey">
```

Comment:
I wonder whether all of the footer content (ie: everything except the dark background) should go inside a fixed-width container? Just feels like, on a screen like this, the "Council Climate League" logo is _very close_ to the left edge of the window:

![Screenshot 2022-01-13 at 15-38-22 SCORING](https://user-images.githubusercontent.com/739624/149360996-4062c4f1-be2d-40ca-8025-b6d80d0391b6.png)

Up to you though!

Reply:
I think it’s made worse by the `max-width` on `.newsletter-wrapper`, which results in it having lots of space to its right side, so the whole footer feels off-centre on wide screens.